cbor.js

```diff
@@ -398,7 +470,7 @@ var obj = { encode: encode, decode: decode };
 
 if (typeof define === "function" && define.amd)
   define("cbor/cbor", obj);
-else if (typeof module !== "undefined" && module.exports)
+else /* istanbul ignore if */ if (typeof module !== "undefined" && module.exports)
```

Comment:
this is a branch that goes uncovered, but is still obviously a common pattern and correct